docs/testing/power-automate-troubleshooting.md

```diff
@@ -14,7 +14,11 @@ Power Automate lets you take your Office Script automation to the next level. Ho
 
 ## Avoid using relative references
 
-Power Automate runs your script in the chosen Excel workbook on your behalf. The workbook might be closed when this happens. Any API that relies on the user's current state, such as `Workbook.getActiveWorksheet`, fails when run through Power Automate. When designing your scripts, be sure to use absolute references for worksheets and ranges.
+Power Automate runs your script in the chosen Excel workbook on your behalf. The workbook might be closed when this happens. Any API that relies on the user's current state, such as `Workbook.getActiveWorksheet`, may behave differently in Power Automate. Some methods will outright fail in a flow.
```

Comment:
"Some methods will outright fail in a flow." is vague to me, but this could be because I'm not very familiar with Power Automate. Consider adding specificity, like "Some methods will fail in a Power Automate flow."